src/OAuth1/Signature/MethodRSASHA1.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * SocialConnect project
+ * @author: Patsura Dmitry https://github.com/ovr <talk@dmtry.me>
+ */
+
+namespace SocialConnect\OAuth1\Signature;
+
+use SocialConnect\Provider\Consumer;
+use SocialConnect\OAuth1\Request;
+use SocialConnect\OAuth1\Token;
+use SocialConnect\OAuth1\Util;
+
+class MethodRSASHA1 extends AbstractSignatureMethod
+{
+    private $private_key;
```

Comment:
I think I remove that one and use the ```applicationSecret```-parameter for that… 'Cause that's what it is in the end…